shared/profile/prove-enter-username.desktop.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React, {Component} from 'react'
+import {Box, Icon, Text, Button, Input, PlatformIcon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Platforms} from '../constants/types/more'
+import type {Props} from './prove-enter-username'
+
+const platformText : {[key: Platforms | 'btc']: {headerText: string, floatingLabelText?: string, hintText?: string}} = {
+  'twitter': {
```

Comment:
can't we make a little function that generates the common case here?

```
const makeCommon = (platform) => ({
headerText: `Prove your ${platform} identity`,
floatingLabelText: `Your ${platform} username`,
})
```

then just

```
'twitter': { ...makeCommon('Twitter') }, 
```